README.md

```diff
@@ -2,3 +2,23 @@
 Documentation for conda-forge.
 
 [![Build Status](https://travis-ci.org/conda-forge/conda-forge.github.io.svg?branch=master)](https://travis-ci.org/conda-forge/conda-forge.github.io)
+
+## Building the docs
+
+The docs are built on travis ci with the `.travis.yml` file at the root of this repository.
+`.travis.yml` uses [conda-execute](https://github.com/conda-tools/conda-execute) to run the `.ci_scripts/update_docs` script.
+To build the docs locally, you should do the following:
+1. `conda env create -f ./.ci_support/environment.yml`
+2. `conda activate conda-forge-docs`
+3. `cd src`
+4. `make html`
+
+## Conda forge dev meetings
+
+A directory containing the minutes from previous dev meetings can be found in /src/minutes or on our website at **insert website link here**.
+These meetings occur every two weeks on Wednesday from 17:30-18:00 UTC.
+A link to the google calendar item can be found [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=dGozZTc1NjhvcmY1cTh1Nm85ODJhZXBibWNfMjAxOTA3MTFUMTUwMDAwWiBlZGlsbEBhbmFjb25kYS5jb20&tmsrc=edill%40anaconda.com).
```

Comment:
The link to the event doesn't work for me.

Reply:
Does this link work? `https://calendar.google.com/event?action=TEMPLATE&tmeid=Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0&tmsrc=scopatz%40gmail.com&scp=ALL`

Reply:
That one works for me. Thanks @scopatz. Thanks for checking the link @xhochy !

Reply:
@scopatz  Also works for me but the time in the link doesn't match the text. This Meeting is shown to me as 18:00-18:30 CEST (which is then 16:00-16:30 UTC).